zerver/lib/zcommand.py

```diff
@@ -32,4 +45,23 @@ def process_zcommands(content: str, user_profile: UserProfile) -> Dict[str, Any]
         ret = dict(msg=msg)
         return ret
 
+    if command == 'mute_topic':
+        linked_stream = possible_linked_stream_names(content)
+        if len(tokens) < 3 or len(linked_stream) != 1:
+            return dict(msg="Usage: /mute_topic #<stream_name> <topic_name>")
+
+        [stream_name] = linked_stream
+        topic = get_topic_from_mute_topic_content(content, stream_name)
```

Comment:
Minor: Reiterating what I said previously or before, `linked_stream[0]` might be simpler?

Reply:
@neiljp `linked_stream` is a set containing only one element. So I can user `list(linked_stream)[0]`, but I thought this looks much more cleaner.